templates/Submissions/view.php

```diff
@@ -254,7 +254,7 @@
                     <?php
                     echo $this->Html->link(
                         _('Browse Files'),
-                        'https://jbez.io500.org/storage/' . $submission->information_list_name . '/' . str_replace('.zip', '', $submission->storage_data),
+                        'https://io500.org/storage/' . $submission->information_list_name . '/' . str_replace('.zip', '', $submission->storage_data),
```

Comment:
Should the base URL also be pulled from Configure::read()?

Reply:
Not necessary, but we could do it if you prefer. Having it defined here will allow people that clone the repository and work on it locally to still have the correct links to the files at the server.